app/pages/sections/catalogue-solutions/order-item/template.njk

```diff
@@ -19,34 +19,39 @@
 
     <div class="nhsuk-grid-row">
       <div class="nhsuk-grid-column-two-thirds">
+        {% if (errors.length > 0) %}
+            {{ bcErrorSummary({
+              errors: errors
+            }) }}
+        {% endif %}
         <h1 data-test-id="order-item-page-title" class="nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-3">{{ title }}</h1>
         <h2 data-test-id="order-item-page-description" class="nhsuk-u-font-size-24 nhsuk-u-margin-bottom-6">{{ description }}</h2>
       </div>
 
-        <form method="post">
-          <div class="nhsuk-grid-column-two-thirds">
-            <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+      <form method="post">
+        <div class="nhsuk-grid-column-two-thirds">
+          <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
 
-            <div class="nhsuk-u-margin-bottom-5">
-              {{ dateField({ question: questions.plannedDate}) }}
-            </div>
+          <div class="nhsuk-u-margin-bottom-5">
+            {{ dateField({ question: questions[0]}) }}
+          </div>
 
           <div class="nhsuk-u-margin-bottom-5">
             <div class="">
-            {{ textField({ 
-              question: questions.quantity,
-              classes: "nhsuk-input--width-10"
-            }) }}
-            {{ viewExpandableSection(questions.quantity.expandableSection) }}
+              {{ textField({ 
+                question: questions[1],
+                classes: "nhsuk-input--width-10"
+              }) }}
+              {{ viewExpandableSection(questions[1].expandableSection) }}
             </div>
 
             <div class="nhsuk-u-margin-bottom-5">
-            {{
-              radiobuttonOptions({
-                question: questions.estimationPeriod
-              })
-            }}
-            {{ viewExpandableSection(questions.estimationPeriod.expandableSection) }}
+              {{
+                radiobuttonOptions({
+                  question: questions[2]
+                })
+              }}
+              {{ viewExpandableSection(questions[2].expandableSection) }}
```

Comment:
😞 preferred what you had before but we are just going to have to live with this for now